2013-12-06T08:39:18.815162+00:00 app[web.1]:     return self.view_functions[rule.endpoint](**req.view_args)
2013-12-06T08:39:18.815311+00:00 app[web.1]:     return self._execute_and_instances(context)
2013-12-06T08:39:18.815311+00:00 app[web.1]:     availableParties = [Combination(c.partyIndex, c.instanceName, c.playerName, c.name, c.className, c.rolename) for c in cur]
2013-12-06T08:39:18.815311+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2353, in __iter__
2013-12-06T08:39:18.778202+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-12-06T08:39:18.778202+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
2013-12-06T08:39:18.815311+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2368, in _execute_and_instances
2013-12-06T08:39:18.815311+00:00 app[web.1]:     result = conn.execute(querycontext.statement, self._params)
2013-12-06T08:39:18.778400+00:00 app[web.1]:     response = self.full_dispatch_request()
2013-12-06T08:39:18.778400+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
2013-12-06T08:39:18.815717+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 874, in _execute_context
2013-12-06T08:39:18.815717+00:00 app[web.1]:     context)
2013-12-06T08:39:18.815311+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 761, in _execute_clauseelement
2013-12-06T08:39:18.815311+00:00 app[web.1]:     compiled_sql, distilled_params
2013-12-06T08:39:18.815717+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1024, in _handle_dbapi_exception
2013-12-06T08:39:18.815717+00:00 app[web.1]:     exc_info
2013-12-06T08:39:18.815717+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 867, in _execute_context
2013-12-06T08:39:18.815717+00:00 app[web.1]:     reraise(type(exception), exception, tb=exc_tb)
2013-12-06T08:39:18.815717+00:00 app[web.1]:     context)
2013-12-06T08:39:18.815717+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 196, in raise_from_cause
2013-12-06T08:39:18.815862+00:00 app[web.1]: ProgrammingError: (ProgrammingError) operator does not exist: character varying = integer
2013-12-06T08:39:18.815862+00:00 app[web.1]: LINE 3: WHERE party_combo.g_spreadsheet_id =  -1 AND party_combo.g_w...
2013-12-06T08:39:18.778605+00:00 app[web.1]:   File "/app/rosterrun.py", line 111, in show_entries
2013-12-06T08:39:18.778400+00:00 app[web.1]:     return self.view_functions[rule.endpoint](**req.view_args)
2013-12-06T08:39:18.815862+00:00 app[web.1]:  'SELECT party_combo.id AS party_combo_id, party_combo.g_spreadsheet_id AS party_combo_g_spreadsheet_id, party_combo.g_worksheet_id AS party_combo_g_worksheet_id, party_combo."partyIndex" AS "party_combo_partyIndex", party_combo."instanceName" AS "party_combo_instanceName", party_combo."playerName" AS "party_combo_playerName", party_combo.name AS party_combo_name, party_combo."className" AS "party_combo_className", party_combo.rolename AS party_combo_rolename \nFROM party_combo \nWHERE party_combo.g_spreadsheet_id = %(g_spreadsheet_id_1)s AND party_combo.g_worksheet_id = %(g_worksheet_id_1)s' {'g_spreadsheet_id_1': -1, 'g_worksheet_id_1': -1}
2013-12-06T08:39:18.815862+00:00 app[web.1]:                                            ^
2013-12-06T08:39:18.815862+00:00 app[web.1]: HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
2013-12-06T08:39:18.778605+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2353, in __iter__
2013-12-06T08:39:18.778605+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 761, in _execute_clauseelement
2013-12-06T08:39:18.778605+00:00 app[web.1]:     compiled_sql, distilled_params
2013-12-06T08:39:18.779006+00:00 app[web.1]:     context)
2013-12-06T08:39:18.779006+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1024, in _handle_dbapi_exception
2013-12-06T08:39:18.779006+00:00 app[web.1]:     exc_info
2013-12-06T08:39:18.779168+00:00 app[web.1]: HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
2013-12-06T08:39:18.779168+00:00 app[web.1]:                                            ^
2013-12-06T08:39:18.814967+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
2013-12-06T08:39:18.814967+00:00 app[web.1]:     return self.wsgi_app(environ, start_response)
2013-12-06T08:39:18.815162+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
2013-12-06T08:39:18.815162+00:00 app[web.1]:     response = self.full_dispatch_request()
2013-12-06T08:39:18.815162+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
2013-12-06T08:39:18.815162+00:00 app[web.1]:     rv = self.dispatch_request()
2013-12-06T08:39:18.815311+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 662, in execute
2013-12-06T08:39:18.815311+00:00 app[web.1]:     params)
2013-12-06T08:39:18.815717+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 324, in do_execute
2013-12-06T08:39:18.815717+00:00 app[web.1]:     cursor.execute(statement, parameters)
2013-12-06T08:39:18.268801+00:00 heroku[router]: at=info method=GET path=/ host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=31ms status=500 bytes=135
2013-12-06T08:38:47.456085+00:00 heroku[router]: at=info method=GET path=/static/style.css host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=10ms service=20ms status=200 bytes=1028
2013-12-06T08:40:35.698448+00:00 heroku[router]: at=info method=GET path=/ host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=9ms status=302 bytes=219
2013-12-06T08:40:35.974001+00:00 heroku[router]: at=info method=GET path=/login host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=6ms service=8ms status=200 bytes=513
2013-12-06T08:40:36.284122+00:00 heroku[router]: at=info method=GET path=/static/style.css host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=11ms status=200 bytes=1028
2013-12-06T08:41:29.123339+00:00 app[web.1]: running with user lisayoung.nz@gmail.com 
2013-12-06T08:41:29.142467+00:00 heroku[router]: at=info method=POST path=/login host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=29ms status=302 bytes=803
2013-12-06T08:42:37.978402+00:00 heroku[router]: at=info method=GET path=/auth_return?code=4/9OFJNqt_eRTtGYMFumvyDVdHpMyS.0gJU9fxwipwegrKXntQAax1y6SiOhQI host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=1672ms status=302 bytes=209
2013-12-06T08:42:38.207834+00:00 app[web.1]: running with user lisayoung.nz@gmail.com 
2013-12-06T08:42:38.207196+00:00 app[web.1]: cannot find gdoc name
2013-12-06T08:42:38.207429+00:00 app[web.1]: show entries
2013-12-06T08:42:38.207660+00:00 app[web.1]: could not find the spreadsheet id
2013-12-06T08:42:38.225572+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
2013-12-06T08:42:38.225572+00:00 app[web.1]:     return self.wsgi_app(environ, start_response)
2013-12-06T08:42:38.225572+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
2013-12-06T08:42:38.225572+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
2013-12-06T08:42:38.233907+00:00 heroku[router]: at=info method=GET path=/ host=rosterrun.herokuapp.com fwd="122.59.245.135" dyno=web.1 connect=2ms service=25ms status=500 bytes=135
2013-12-06T08:42:38.225572+00:00 app[web.1]:     response = self.make_response(self.handle_exception(e))
2013-12-06T08:42:38.225572+00:00 app[web.1]: 2013-12-06 08:42:38 [7] [ERROR] Error handling request
2013-12-06T08:42:38.225572+00:00 app[web.1]: Traceback (most recent call last):
2013-12-06T08:42:38.225572+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 131, in handle_request
2013-12-06T08:42:38.225572+00:00 app[web.1]:     respiter = self.wsgi(environ, resp.start_response)
2013-12-06T08:42:38.225572+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-12-06T08:42:38.225757+00:00 app[web.1]:     rv = self.handle_user_exception(e)
2013-12-06T08:42:38.225757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
2013-12-06T08:42:38.225757+00:00 app[web.1]:     rv = self.dispatch_request()
2013-12-06T08:42:38.225757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
2013-12-06T08:42:38.225757+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-12-06T08:42:38.225757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
2013-12-06T08:42:38.225757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
2013-12-06T08:42:38.225757+00:00 app[web.1]:     response = self.full_dispatch_request()
2013-12-06T08:42:38.225757+00:00 app[web.1]:     return self.view_functions[rule.endpoint](**req.view_args)
2013-12-06T08:42:38.225906+00:00 app[web.1]:   File "/app/rosterrun.py", line 124, in show_entries
2013-12-06T08:42:38.225906+00:00 app[web.1]:     (g_s_id, g_w_id) = testConnectToSpreadsheetsServiceOAuth(credentials, session['doc'])
2013-12-06T08:42:38.225906+00:00 app[web.1]:     feed = gd_client.GetSpreadsheets(query=q)
2013-12-06T08:42:38.225906+00:00 app[web.1]:   File "/app/scheduler.py", line 159, in testConnectToSpreadsheetsServiceOAuth
2013-12-06T08:42:38.225757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
2013-12-06T08:42:38.225906+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gdata/client.py", line 640, in get_feed
2013-12-06T08:42:38.225906+00:00 app[web.1]:     desired_class=desired_class, **kwargs)
2013-12-06T08:42:38.225906+00:00 app[web.1]:     uri=uri, auth_token=auth_token, http_request=http_request, **kwargs)
2013-12-06T08:42:38.225906+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gdata/spreadsheets/client.py", line 78, in get_spreadsheets
2013-12-06T08:42:38.225906+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gdata/client.py", line 267, in request
2013-12-06T08:42:38.226306+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/atom/client.py", line 101, in request
2013-12-06T08:42:38.226306+00:00 app[web.1]:     http_request.uri.query[name] = str(value)
2013-12-06T08:42:38.226306+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gdata/service.py", line 1718, in __str__
2013-12-06T08:42:38.226306+00:00 app[web.1]:     return self.ToUri()
2013-12-06T08:42:38.226306+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gdata/service.py", line 1715, in ToUri
2013-12-06T08:42:38.225906+00:00 app[web.1]:     **kwargs)
2013-12-06T08:42:38.226306+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/atom/service.py", line 584, in BuildUri
2013-12-06T08:42:38.226306+00:00 app[web.1]:     parameter_list = DictionaryToParamList(url_params, escape_params)
2013-12-06T08:42:38.226306+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/atom/service.py", line 551, in DictionaryToParamList
2013-12-06T08:42:38.226453+00:00 app[web.1]:     if ' ' in s:
2013-12-06T08:42:38.226306+00:00 app[web.1]:     for param, value in (url_parameters or {}).items()]
2013-12-06T08:42:38.226453+00:00 app[web.1]: TypeError: argument of type 'NoneType' is not iterable
2013-12-06T08:42:38.226306+00:00 app[web.1]:     return atom.service.BuildUri(q_feed, self)
2013-12-06T08:42:38.226453+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/urllib.py", line 1286, in quote_plus
